manta-js/README.md

```diff
@@ -1,280 +1,27 @@
-- `toPublicSend(asset, amount, polkadotSigner, polkadotAddress)`
-- `publicTransfer(asset, amount, destinationAddress, polkadotSigner, polkadotAddress)`
-
-> This example assumes the `polkadotAddress` already has associated public funds.
-
-### To Private
-
-This example converts 10 public DOL tokens to 10 private DOL tokens.
-
-```javascript
-// DOL token
-const assetId = new BN("1");
-const amount = new BN("10000000000000000000");
-
-// Sync with most recent ledger state.
-await privateWallet.initialWalletSync();
-
-// Get private address
-const privateAddress = await privateWallet.getPrivateAddress();
-
-// Get private balance of DOL for given private address
-const privateBalance = await privateWallet.getPrivateBalance(assetId);
-
-// Privatize 10 DOL to 10 pDOL
-await privateWallet.toPrivateSend(assetId, amount, polkadotSigner, polkadotAddress);
-
-// Sync to get latest data after the transaction and check that it was successful.
-await privateWallet.walletSync();
-
-// The private balance of pDOL should be incremented by 10 units.
-const newPrivateBalance = await mantaSdk.getPrivateBalance(assetId);
-```
-
-### Private Transfer
-
-This example transfers 10 private private pDOL to another address.
-
-```javascript
-// DOL token
-const assetId = new BN("1");
-const amount = new BN("10000000000000000000");
-
-// Sync with most recent ledger state.
-await privateWallet.initialWalletSync();
-
-// Get private address
-const privateAddress = await privateWallet.getPrivateAddress();
-
-// Private Transfer of 10 pDOL to another private address
-const examplePrivateAddress = "3UG1BBvv7viqwyg1QKsMVarnSPcdiRQ1aL2vnTgwjWYX";
-await privateWallet.privateTransferSend(assetId, amount, examplePrivateAddress, polkadotSigner, polkadotAddress);
-
-// Sync to get latest data after transaction and check that it was successful.
-await privateWallet.walletSync();
-
-// The private balance of pDOL should decrease by 10 units.
-const newPrivateBalance = await privateWallet.getPrivateBalance(assetId);
-```
-
-### To Public
-
-This example converts 5 private pDOL to 5 public DOL.
+## How to build
 
-```javascript
-// DOL token
-const assetId = new BN("1");
-const amount = new BN("5000000000000000000");
-
-// Sync with most recent ledger state.
-await privateWallet.initialWalletSync();
-
-// Get private address
-const privateAddress = await privateWallet.getPrivateAddress();
-
-// Get private balance of DOL for given private address
-const privateBalance = await privateWallet.getPrivateBalance(assetId);
-
-// Convert 5 pDOL back to DOL
-await privateWallet.toPublicSend(assetId, amount, polkadotSigner, polkadotAddress);
-
-// Sync to get latest data after transaction and check that it was successful.
-await privateWallet.walletSync();
-
-// The private balance of pDOL should decrease by 5 units.
-const newPrivateBalance = await privateWallet.getPrivateBalance(assetId);
+``` sh
+cd ./package
+yarn
+yarn build-browser
```

Comment:
Is this library ever going to be used outside of the browser going forward? If not, we should maybe get rid of build commands for node.

Reply:
I use node for mocha tests and same with georgi for manta CI, so I think it would be good to keep unless there is a reason to remove node... although I'm not sure how accessing extension from node environment will work (if its possible)

Reply:
I agree with @ferrell-code 

But the node side has nothing to do with the extension. It may be used as a pure server to provide services, but this is also my assumption, and see if there is a need for this in the future.